e3db-crypto-android/src/main/java/com/tozny/e3db/android/KeyAuthentication.java

```diff
@@ -190,9 +291,10 @@ public String toString() {
   /**
    * Creates an instance requiring lock screen authentication (using the default
    * timeout of {@link #DEFAULT_LOCK_SCREEN_TIMEOUT}).
-   *
+   * <p>
```

Comment:
what's with the paragraph tags in the comments?

Reply:
I ran IntelliJ's autoformatter and <p> is the canonical way to do paragraph breaks in javadoc.